exercises/practice/sublist/.meta/Sublist.example.ps1

```diff
@@ -0,0 +1,59 @@
+Enum Sublist {
```

Comment:
I think an append could be needed here, to tell the user that they will have to define an enum. Nothing long is needed, maybe a link to the Powershell docs about Enum and telling the user that is required is more or less all that is required.

For an append the path should be: `.docs/instructions.append.md`